ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-v0.15.2__4.5.yaml

```diff
@@ -68,6 +68,14 @@ images:
       - destination_dir: .
         source_path: /go/bin/autoscaler
   to: knative-serving-autoscaler
+- dockerfile_path: openshift/ci-operator/knative-images/migrate/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/migrate
+  to: knative-serving-storage-version-migration
```

Comment:
I like the name (`knative-serving-storage-version-migration`) to indicate this comes from the vendor'ed serving bits :+1: